meetings/2025-02-11

```diff
@@ -1,82 +0,0 @@
-Presto TSC Meeting
```

Comment:
I don't know why this does not show as a rename, and instead shows this as a delete + new file. Anyway, I've kept the agenda the same as before, I just added markdown formatting.